include/drivers/interrupt_controller/gic.h

```diff
@@ -220,6 +233,12 @@
 /*
  * Common Helper Constants
  */
+#define GIC_SGI_INT_BASE		0
+#define GIC_PPI_INT_BASE		16
+
+#define gic_is_sgi(intid)		(((intid) > GIC_SGI_INT_BASE) && \
+					 ((intid) < GIC_PPI_INT_BASE))
```

Comment:
```suggestion
#define gic_is_sgi(intid)		(((intid) >= GIC_SGI_INT_BASE) && \
					 ((intid) < GIC_PPI_INT_BASE))
```

p.s. These macros should probably be named with all uppercase; at least, that is the convention used by Zephyr.

Reply:
Thanks.  Ack.